defaults/main.yml

```diff
@@ -18,6 +18,10 @@ wimpy_application_pre_commands: []
 wimpy_application_post_commands: []
 wimpy_application_additional_units: []
 
+# CloudWatch logs
+wimpy_aws_cloudwatch_enabled: True
```

Comment:
I disagree with @Hyunk3l, if you have a small project but you chose deploy using Wimpy so that you will use a bunch of AWS resources like ec2 (instances, elb, asg), CloudFormation, Route53 or ECR, don't you have 0.02 cents more to use cloudwatch? :D

+1 to enabled by default.